### ISSUE TITLE
REQUEST: can you please add virtual navigation

### ISSUE THREAD

**Issue by deep5050** [Genymobile/scrcpy#736]
can you please add an option for virtual navigation ?? i'm using gesture navigation mode on my MIUI 10 .. it's very hard to draw those gestures with mouse or touch-pad and i don't want to switch from full-screen mode on my phone. thanks in advance :)

**Comment by Lucaacer** [comment]
You can try

CTRL + b for back button

CTRL + h for home button

CTRL + m for menu/recent apps

CTRL + n for notification panel

I find those commands very handy and I just miss the swipe commands (up, right and left) which are mandatory for some apps.

**Comment by glacjay** [comment]
@Lucaacer not really. I'm using mouse to control scrcpy most time, and these shortcuts are a little hard to push using only left hand 😂️

**Comment by rom1v** [comment]
Right-click for BACK
Middle-click for HOME

**Comment by rom1v** [comment]
Ref #7

**Comment by glacjay** [comment]
Thanks, but what about ctrl-m?

**Comment by dcoletto** [comment]
Just my 2 cents. I would totally add some virtual navigation button, like the official android emulator does. I have a logitech mouse and my middle click is remapped, so I can't press the home button easily. Even using a touchpad or the apple magic mouse I'm not really sure you can "middle click". So the 3/4 virtual nav button would (maybe with an option to enable/disable them) would be really appreciated. If you're interested I can write the code on my own and make a pull request, but I won't do that if that's not something you're considering to merge (honestly I don't want to maintain a fork, so I won't do that if that will not be merged)
(PS: I'll be able to work on that in 2 weeks or so anyway)
